node/rpc-api/src/lib.rs

```diff
@@ -53,7 +53,7 @@ pub static ACCESS_MAP: Lazy<HashMap<&str, Access>> = Lazy::new(|| {
 
     // Wallet API
     access.insert(wallet_api::WALLET_BALANCE, Access::Write);
-    access.insert(wallet_api::WALLET_DEFAULT_ADDRESS, Access::Write);
+    access.insert(wallet_api::WALLET_DEFAULT_ADDRESS, Access::Read);
```

Comment:
@lemmih Is this change expected or only for testing?

Reply:
Expected, I think. Should be okay to have the default wallet address be publically readable.